### DISCUSSION TITLE
How to use yaml prompts in wildcards

### DISCUSSION THREAD

**Discussion by chouti** [adieyal/sd-dynamic-prompts#560] [Q&A]
Forgive me if I asked a dummy question, but I've confused with the yaml file prompts.

Firstly I've downloaded a wildcard dataset, which contains both txt and yaml files. I know how to use txt files. But within yaml file, I've got contents like below:
```
'SFW, 1girl, solo focus, full body, __Art Styles__, <[Camera Angle]>, <[Emotions]>,  <[Pose]>, {<[HighSFW|MedSFW|LowSFW][Fullbody]>|<[Torso][--Sexywear]>, <[Waist][--Sexywear]>}, {20%<[Modifier][SFW]>, |}<[Age][Female][Adult]>, <[Hair Length]>, <[Hair Style]>, <[Hair Color Types]>':
 Description:
  - "A fully randomized SFW Female character."
 Tags:
  - Preset
  - AIOGirl
  - SFW
```
My question would be how do I use such prompt in A1111 webUI?

I've tried input
`SFW, 1girl, solo focus, full body, __Art Styles__, <[Camera Angle]>, <[Emotions]>,  <[Pose]>, {<[HighSFW|MedSFW|LowSFW][Fullbody]>|<[Torso][--Sexywear]>, <[Waist][--Sexywear]>}, {20%<[Modifier][SFW]>, |}<[Age][Female][Adult]>, <[Hair Length]>, <[Hair Style]>, <[Hair Color Types]>`
directly into the prompt text box, but seems A1111 webUI threat  `<[Camera Angle]>` as a token, I supposed it should iterate some sort of list just like `__Art Styles__`
Can anyone guided me, if I'm missing something?

**Comment by akx** [comment]
That doesn't seem to be a dataset compatible with this extension – the syntax is for some other wildcard tool.

**Comment by adieyal** [comment]
Here is an example that might be useful: 
https://github.com/adieyal/sd-dynamic-prompts/blob/main/collections/publicprompts.yaml
